main-project/src/meetings/meetings.service.ts

```diff
@@ -1,11 +1,18 @@
 import { Injectable, BadGatewayException } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
+import { GuestMembersRepository } from 'src/members/repository/guest-members.repository';
+import { HostMembersRepository } from 'src/members/repository/host-members.repository';
 import { CreateMeetingDto } from './dto/createMeeting.dto';
 import { UpdateMeetingDto } from './dto/updateMeeting.dto';
 import { Meetings } from './entity/meeting.entity';
 import { MeetingInfoRepository } from './repository/meeting-info.repository';
 import { MeetingRepository } from './repository/meeting.repository';
 
+export interface meetingResponseInfo {
```

Comment:
인터페이스의 네이밍 규칙은 PascalCase를 따르는 것이 일반적이에요.
즉, 첫 글자는 대문자가 일반적입니다 :D

```suggestion
export interface MeetingResponseInfo {
```

추가로, ```info``` 라는 키워드를 식별자에 많이 붙인 것 같은데 최대한 붙이지 않는 것이 더 좋은 관례에요.
```info```를 굳이 붙이지 않더라도 *정보* 라는 의미가 암묵적으로 식별자에 담겨 있어요.

이와 비슷한 케이스로 ```data```, ```list```, ```str``` 이런게 있을 수 있어요.
```data```는 info와 같은 경우에요.

```list```와 ```str```은 식별자에 타입을 명시한 경우에요.
타입이 없는 언어의 경우 식별자로 타입을 표현해주기도 했는데, 지금은 그러지 않는게 추세가 됐어요.
또한, 사용하고 있는 언어가 TS기에 더욱 더 식별자에 타입을 명시해줄 필요가 없겠지요.
그런 케이스는 없었지만, 여러 예시를 보여주기 위해 말씀 드려봅니다 :D

Reply:
추가로 인터페이스는 별도 폴더에서 따로 관리해주세요.
어느 service에는 인터페이스가 있고, 어디에는 없고 이러면 관리하기가 어려워집니다. 
파일이 많아질 수록요 :D